### ISSUE TITLE
Is there a way to decompose AE effect to make it exportable as JSON?

### ISSUE THREAD

**Issue by sneko** [airbnb/lottie-web#2700]
Hi @bodymovin ,

As for Christian (https://github.com/airbnb/lottie-web/issues/1991), I faced the issue of my animation not being exported due to "CC Bender" being not supported. I know this effect is not supported by Bodymovin... but....

I'm wondering if there is kind of an automated workaround? Something like:
* I use "CC Bender" for a 2 seconds animation
* It cannot be exported through Bodymovin
* But a "AE plugin" (?) could split the 2 seconds animation into 60 frames (if we say 30fps) (it's no longer a "CC Bender" effect but only 60 frames with all different shapes that represent the whole animation)
* And Bodymovin can export?

I don't know if it sounds possible, it's probably not optimised since you would have a lot of different "duplicated paths", but at least for short animation it would make it renderable.

Thank you,

EDIT: just found this example https://aydinuzmez.medium.com/how-to-make-image-sequences-with-lottie-in-android-easy-66af61c221ea with exported PNG sequence. Still wondering about sequencing with SVG to keep best quality/size...

**Comment by bodymovin** [comment]
hi, I don't think CC bender would translate easily into bezier curves, since it's operating at the pixel level and the result can be any kind of curve.
You can try some sort of rotoscoping but it ill probably end up as a huge file size.

**Comment by sneko** [comment]
@bodymovin indeed, if based on pixel level there will be no magic trick 👍 

Currently I did the PNG sequence, and embedded them in a Lottie export. It sounds more like a "video" for now (not using Lottie feature), but that's the workaround I use until we do a real animation simulating the "CC Bender".

Another solution would be to use a web library to directly distort (mimic a "CC Bender" effect), but from what I saw... it's like needed to go deep in programming details... and I'm afraid to waste too much time on it. Having a not responsive animation will do the trick for now.

Thank you for answering, and for your plugin 😀 !
